web/ASC.Web.Common/src/components/headline/README.md

```diff
@@ -1,29 +1,29 @@
-# Header
+# Headline
 
-Component that displays header text
+Component that displays Heading text with custom styles
 
-### Usage
+## Usage
```

Comment:
Use the basic markdown readme conventions: 
- Component name - #
- Category - ###
- Subcategory - ####